tc/core/flags.cc

```diff
@@ -47,6 +47,10 @@ DEFINE_string(
     llvm_flags,
     "-std=c++11 -O3 -ffast-math",
     "compiler flags to set when llvm is used");
+DEFINE_string(
+    nvcc_flags,
+    "-std=c++11 -ptx -DNVRTC_CUB=1 --use_fast_math",
```

Comment:
Why is `NVRTC_CUB=1` necessary for _nvcc_ compilation?  The modified CUB was meant to circumvent nvrtc's inability to load system headers, which should not be a problem for nvcc.